bodhi/server/models.py

```diff
@@ -2583,13 +2584,41 @@ def get_title(self, delim=' ', limit=None, after_limit='…'):
                 number. If ``None`` (the default), no limit is used.
             after_limit (str): If a limit is set, use this string after the limit is reached.
                 Defaults to '…'.
+            beautify (bool): If provided the returned string will be human readable.
+                3 or more builds will take the form "package1, package, 2 and XXX more"
+            nvr (bool): If specified the title will include name, version and
+                release information in package labels.
+            amp (bool): If specified it will replace the and with and
+                &amp; html entity
         Returns:
             str: A title for this update.
         """
-        all_nvrs = [x.nvr for x in self.builds]
-        nvrs = all_nvrs[:limit]
-        builds = delim.join(sorted(nvrs)) + (after_limit if limit and len(all_nvrs) > limit else "")
-        return builds
+        if beautify:
+            if self.display_name:
+                return self.display_name
+
+            def build_label(build):
+                return build.nvr if nvr else build.package.name
+
+            if len(self.builds) > 2:
+                title = ", ".join([build_label(build) for build in self.builds[:2]])
+
+                if amp:
+                    title += ", &amp; "
```

Comment:
This has become simply "&" since this PR was opened.

I'm curious why this isn't being caught by the CI system though :(

Reply:
OK I'll make that change.